src/components/DocumentationTopic.vue

```diff
@@ -297,15 +293,15 @@ export default {
     shouldShowLanguageSwitcher: ({ objcPath, swiftPath }) => objcPath && swiftPath,
     hideSummary: () => getSetting(['features', 'docs', 'summary', 'hide'], false),
     enhanceBackground: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
-    technologies({ modules = [], technology }) {
+    technologies({ modules = [] }) {
       const technologyList = modules.reduce((list, module) => {
         list.push(module.name);
         return list.concat(module.relatedModules || []);
       }, []);
-      // show modules if page belongs to/require multiple technologies
-      // or if name doesn't match root of page
-      return technologyList.length === 1
-        && technologyList[0] === technology.title ? [] : technologyList;
+      // only show badges for technologies when there are multiple
```

Comment:
Sorry I forgot where we landed with this. Should we also check if technologyList[0].relatedModules exist? In that case, technically we also have multiple technologies.

Reply:
Yep. I didn't delete that (line 299)

Reply:
Yea sorry I confused myself 😅